### ISSUE TITLE
Exception details plz check

### ISSUE THREAD

**Issue by ghost** [infinity-next/infinity-next#330]
I'm not sure what is wrong. I followed step by step, with no errors.

![image](https://user-images.githubusercontent.com/89582987/130959111-7a7fbd02-7c86-4b9d-8a71-9b2cfe9266f2.png)

**Comment by ghost** [comment]
And also, there's this:
![image](https://user-images.githubusercontent.com/89582987/130959229-21b5b5b1-1f27-41fc-aa77-5b3c224f7226.png)

**Comment by jaw-sh** [comment]
homie I haven't worked on this in over a year, but from the looks of it, you've not ran Mix, which requires command line access and JavaScript.

https://laravel.com/docs/6.x/mix

This is how all static asset compilations are done.

**Comment by ghost** [comment]
npm install, is that right? Because if so, I've done so already. Like said before, I've done all the steps and in Debian 10

**Comment by jaw-sh** [comment]
No, it's like npm mix or something. You actually have to run a command to compile the assets. I cannot walk you through this.
